portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,26 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import com.google.gson.Gson;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+    ArrayList<String> messages = new ArrayList<String>();
+
+    public DataServlet() {
+        messages.add("This is the first message.");
+        messages.add("This is the second message!");
+        messages.add("This is the third message!!");
+    }
```

Comment:
Perhaps this can have a better indentation (2 spaces).
Google HTML/CSS style guide could be helpful for you: https://google.github.io/styleguide/htmlcssguide.html